src/components/Permissions/DataList.spec.jsx

```diff
@@ -0,0 +1,29 @@
+import { render } from '@testing-library/react'
+import React from 'react'
+import DataList from './DataList'
+
+jest.mock('cozy-ui/transpiled/react', () => {
+  return { useI18n: () => ({ t: x => x }) }
+})
```

Comment:
nit: Instead of mocking useI18n each time for each component, you can create a Provider for your test. We call it `AppLike`. 

No need to be complicated at first, you can just add a `<I18n>` provider https://github.com/cozy/cozy-banks/blob/master/test/AppLike.jsx#L16-L22 

and then return it https://github.com/cozy/cozy-banks/blob/master/test/AppLike.jsx#L36

And when you test your component, instead of doing: 
```
    const { queryByText, container } = render(<DataList />)
```

you can do: 
```
    const { queryByText, container } = render(<AppLike><DataList /></AppLike>)
```

With that, no need to mock use18n anymore. 

This pattern will be useful to mock cozy-client store and else. We have it almost in all our apps.